conf/iCubFindDependencies.cmake

```diff
@@ -64,7 +64,7 @@ checkandset_dependency(IPOPT)
 checkandset_dependency(OpenCV)
 checkandset_dependency(Qt5)
 
-set(MINIMUM_REQUIRED_icub_firmware_shared_VERSION 1.39.1)
+set(MINIMUM_REQUIRED_icub_firmware_shared_VERSION 1.41.0)
```

Comment:
Hi @ale-git, pls set it to be 1.40.1 so that it matches what in here: https://github.com/robotology/icub-firmware-shared/pull/99#discussion_r1846898167